pkg/webui/constants/sensitive-data.js

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default ['primary_email_address', 'eui', 'apiKeys', 'dev_eui', 'join_eui']
```

Comment:
@htdvisser can you share a braindump of what other properties you deem to be sensitive?

```suggestion
export default ['primary_email_address', 'email', 'email_address', 'contact_info', 'eui', 'apiKeys', 'api_keys', 'dev_eui', 'join_eui', 'password', 'keys', 'f_nwk_s_int_key', 's_nwk_s_int_key', 'nwk_s_enc_key', 'app_s_key', 'attributes']
```

It would be great if it was possible to define regexps also, e.g. `/key/`.

Reply:
Sentry is compliant with GDPR and other data protection laws, so the goal is not necessarily to redact all privacy-sensitive information. The goal is to redact *secret* information.

I think it's good to make this a regex indeed, then we can filter out:

- equal `password` or end with `.password` or `_password`
- equal `key` or end with `.key` or `_key`
- equal `secret` or end with `.secret` or `_secret`

I think it's a bad idea to redact IDs or EUIs, because that would prevent us from adequately debugging.